examples/index.html

```diff
@@ -1,17 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
+  <meta charset="UTF-8" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+
   <title>Cookie Consent Manager</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
```suggestion
  <meta name="viewport" content="width=device-width, initial-scale=1">
```

> As it is a void element, the start tag must be present and the **end tag must not be present**.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta


(also for the other link/meta tags below and above)

Reply:
> 6. Then, if the element is one of the void elements, or if the element is a foreign element, then there **may be a single U+002F SOLIDUS character (/)**. This character has no effect on void elements, but on foreign elements it marks the start tag as self-closing.

https://html.spec.whatwg.org/multipage/syntax.html#start-tags

🤔

Reply:
> This character has no effect on void elements

Soo, why should we have something which has no effect? :)  Both is valid, thats true, but for consistency - we use `<br>`, `<hr>` etc. in html5, so why this should be an exception?

Reply:
I've been always used to writing trailing `/` everywhere, `<br />` or `<hr />` being no exception. In React projects, ESlint gives me no other choice anyway.

Generally speaking, when there are more possible ways of syntax in a code, I prefer the one which is easier to read and understand, even if it means more writing. To me, trailing slash denotes a self-closing element, as opposed to elements with starting and ending tags, and I'm able to recognize it even without reading actual tag names.

Reply:
Understand your point and read it carefully, but still don't agree its valid for this (eg. this is not XML like JSX). So I don't think there is any benefit in doing thing differently than what is currently the de-facto standard / mostly used. From quick research, the no-slash version is also the usual convention in LMC products. So having example with different conventions will not make it easier for anyone.

So let move on with this - and go with the short version and no closing tags please :)

Reply:
(Or maybe just don't spend time on this now and let it be 🎶 - I will do some more cleanups in #37, so I can do it altogether...)